src/Components/DataEntry/TextField.vue

```diff
@@ -0,0 +1,250 @@
+		<input
+			:id="computedId"
+			class="c-textfield--input"
+			:value="value"
+			:type="type"
+			:placeholder="placeholder"
+			:name="computedId"
+			:label="label"
+			:align="align"
+			:focus="focus"
+			:readonly="readOnly"
+			:disabled="disabled"
+			:error="error"
+			:style="[computedAlign]"
+			:class="[computedLined, computedError]"
+			v-bind="$attrs"
+			@input="$emit('input', $event.target.value)"
+			v-on="$listeners"
+		/>
+		<label v-if="label !== ''" :for="computedId" class="c-textfield--label">{{ label }}</label>
+		<Typography v-if="error" type="caption2" color="red400" element="p" class="c-textfield--message">
+			<slot name="errorMessage" />
+		</Typography>
+	</div>
+</template>
+
+<script>
+import Typography from '@/src/Elements/Core/Typography/Typography';
+export const textfieldTypes = ['text', 'number', 'password', 'email', 'tel', 'url'];
+export const textfieldAligns = ['left', 'center', 'right'];
+
+export default {
+	name: 'TextField',
+	inheritAttrs: false,
+	props: {
+		type: {
+			type: String,
+			default: 'text',
+			validator(value) {
+				const isValid = textfieldTypes.indexOf(value) !== -1;
+				if (!isValid) {
+					console.error(`${value} is not a valid name of the textfield type`);
+				}
+				return isValid;
+			},
+		},
+		value: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		placeholder: {
+			type: String,
+			default: '내용을 입력하세요.',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		label: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		filled: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		align: {
+			type: String,
+			default: 'left',
+			validator(value) {
+				const isValid = textfieldAligns.indexOf(value) !== -1;
+				if (!isValid) {
+					console.error(`${value} is not a valid name of the textfield align`);
+				}
+				return isValid;
+			},
+		},
+		focus: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		readOnly: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		disabled: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		error: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+	},
+	computed: {
+		sync_value: {
+			get() {
+				return this.value;
+			},
+			set(val) {
+				this.$emit('update:value', val);
+			},
+		},
+		computedLined() {
+			if (this.filled) {
+				return 'outlined';
+			} else {
+				return 'underlined';
+			}
+		},
+		computedId() {
+			return this.id || `textField-${this._uid}`;
+		},
+		computedLabelStyle() {
```

Comment:
```suggestion
		computedLabel() {
		    return { label: this.label !== '' };
		}
```
이렇게 하면 어떨까요?
:class인데 style이라는 명칭을 쓰면 :style같이 느껴지네요.